main.py

```diff
@@ -4,9 +4,9 @@
 import uuid
 from pathlib import Path
 
-from quest import event, external_event
+from quest import event
 from quest.workflow import WorkflowManager, JsonEventSerializer, JsonMetadataSerializer, \
-    StatelessWorkflowSerializer
+    StatelessWorkflowSerializer, signal_event
```

Comment:
`event` and `signal_event` should probably come from the same file.

Reply:
Yeah I forgot that was declared in the __init__.py and had to change it there